packages/presentation_layer/lib/app/screens/login/login.dart

```diff
@@ -0,0 +1,72 @@
+import 'package:flutter/material.dart';
+import 'package:presentation_layer/app/bloc/bloc_data.dart';
+import 'package:presentation_layer/app/bloc/bloc_state.dart';
+import 'package:presentation_layer/app/screens/login/login_bloc/login_bloc.dart';
+import 'package:presentation_layer/app/screens/login/login_bloc/login_data.dart';
+import 'package:presentation_layer/utils/helpers/positioned_helper.dart';
+
+class Login extends StatefulWidget {
+  const Login({Key? key}) : super(key: key);
+
+  @override
+  State<Login> createState() => _LoginState();
+}
+
+class _LoginState extends BlocState<Login, LoginBloc> {
+  @override
+  void initState() {
+    super.initState();
+    bloc.initState();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: SafeArea(
+        child: StreamBuilder<BlocData>(
+            stream: bloc.dataStream,
+            builder: (context, snapshot) {
+              final BlocData? blocData = snapshot.data;
+              final bool? screenLoadStatus = blocData?.isLoading;
+              final LoginData? screenData = blocData?.data;
+
+              return Padding(
+                padding: const EdgeInsets.symmetric(horizontal: 20),
+                child: Column(
+                  children: [
+                    TextField(
+                      onChanged: (value) => bloc.onChange(
+                          inputValue: value, inputType: TypeOfInput.login),
```

Comment:
seems to be better to change onChange method to 2 methods - setLogin and setPassword